config/routes.rb

```diff
@@ -6,6 +6,11 @@
   root to: "pages#home"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :educations
+  resources :favorites, only: %i[index]
+
+  resources :universities do
+    resources :favorites, only: %i[create destroy]
```

Comment:
there's already universities path below. Can the favorites path be put under that universities path?